themes/src/main/resources-community/theme/base/admin/messages/admin-messages_pt_BR.properties

```
@@ -883,23 +883,23 @@ key-password.tooltip=Senha para acessar a chave privada no certificado.
 store-password.tooltip=Senha para acessar o arquivo em si.
 jwt-import.key-alias.tooltip=Alias do arquivo para o seu certificado.
 registrationAccessToken.tooltip=O token de acesso para registro provê acesso aos cliente para o serviço de registro cliente.
-scope-param-required.tooltip=Este role somente será concedido se os parâmetros de escopo com os nomes dos roles forem utilizados durante a autorização/solicitação de token.
-composite-roles.tooltip=Quando este role é associado/removido de um usuário, qualquer role associado com ele também será adicionado/removido implicitamente.
-composite.associated-realm-roles.tooltip=Roles de nível de realm associados com este role composto.
-composite.available-realm-roles.tooltip=Roles de nível de realm disponíveis para este role composto.
-available-roles.tooltip=Roles para este cliente que você pode associar a este role composto.
-scope.available-roles.tooltip=Roles do Realm que podem ser associados a este escopo.
-service-account.available-roles.tooltip=Roles do Realm que podem ser associados a contas de serviço.
-client.associated-roles.tooltip=Roles do Cliente associados a este role composto.
+scope-param-required.tooltip=Esta fun\u00E7\u00E3o somente será concedida se os parâmetros de escopo com os nomes dos fun\u00E7\u00F5es forem utilizadas durante a autorização/solicitação de token.
```

```suggestion
scope-param-required.tooltip=Esta fun\u00E7\u00E3o somente será concedida se os parâmetros de escopo com os nomes das fun\u00E7\u00F5es forem utilizados durante a autorização/solicitação de token.
```
I believe the word "utilizados" refers to the parameters, not the roles.